<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Choose</class>
 <widget class="QWidget" name="Choose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pbt2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选好了</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/lenovo/Desktop/res/选好.png</normaloff>C:/Users/lenovo/Desktop/res/选好.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_output">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>351</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>推荐路线：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_route">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>291</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>11</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="ptb_clear">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>重新选</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/lenovo/Desktop/res/重选.png</normaloff>C:/Users/lenovo/Desktop/res/重选.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>起点：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptb">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>111</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox_3">
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>3 东门</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox_1">
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1 西操</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBox_2">
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2 家属区</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBox_4">
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>4 思源楼</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="checkBox_5">
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>5 明湖</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>141</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox_8">
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>8 东区一教</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox_6">
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>6 逸夫楼</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBox_9">
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>9 交大公交站</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBox_7">
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>7 南门</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="checkBox_10">
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>10 学生活动中心</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>121</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox_13">
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>13 嘉园</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBox_14">
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>14 校医院</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBox_12">
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>12 天佑会堂</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox_11">
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>11 图书馆</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="checkBox_15">
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>15 麦当劳</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
    </font>
   </property>
   <property name="text">
    <string>（不添加则默认所选景点中序号最小的为起点）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
